src/functionalTest/resources/scenarios/RIA-3453-make-an-application-notification-home-office-apc-after-listing.json

```diff
@@ -0,0 +1,105 @@
+{
+  "description": "RIA-3453 Send make an application notification home office APC after listing",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "HomeOfficePOU",
+    "input": {
+      "id": 34530,
+      "eventId": "makeAnApplication",
+      "state": "prepareForHearing",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "currentCaseStateVisibleToHomeOfficeAll":"prepareForHearing",
+          "currentCaseStateVisibleToLegalRepresentative": "prepareForHearing",
+          "listCaseHearingCentre": "taylorHouse",
+          "ariaListingReference": "PA/12345/2019",
+          "makeAnApplications": [
+            {
+              "id": "1",
+              "value": {
+                "type":"withdraw",
+                "details":"",
+                "evidence":[],
+                "applicant":"",
+                "date":"{$TODAY}",
+                "decision":"GRANTED",
```

Comment:
what happens if the decision is REFUSED?

Reply:
this scenario is handled in 3454 functional tests where the decision is important, IN 3453, we don't use the decision

Reply:
hmm, why it is GRANTED if the decision is not done yet? same like other fields:
```
                "decisionReason":"No Reason given",
                "decisionDate":"{$TODAY}",
                "decisionMaker": "Judge"
```